src/components/AssetClassification/FeeDestination.tsx

```diff
@@ -1,12 +1,13 @@
-import { FunctionComponent, useState } from "react";
+import { FunctionComponent, useEffect, useState } from "react";
 import styled from "styled-components";
-import { FeeDestination } from "../../models";
+import { EntityDetail, FeeDestination, newEntityDetail } from "../../models";
 import { RemoveButton } from "../Button";
 import { InputOrDisplay } from "../Input";
+import { EntityDetailDisplay } from "./EntityDetailDisplay";
 
 const FeeDestinationWrapper = styled.div`
     display: grid;
-    grid-template-columns: 1fr 1fr auto;
```

Comment:
This auto column was for keeping the remove button to the right when in edit mode, might need to put another wrapper with this grid layout around the address/fee amount/EntityDetailDisplay components, and then one around that and the RemoveButton that is a display:flex or something to keep that button in a nice spot

Reply:
Followed your multi-wrapper approach and I think it looks much nicer now - updated the example image at the top to show the new version.  I waffled between having the delete button on the left or the right, but I think it makes most sense on the left because then all deletion options for things are left-justified and addition options are right-justified for maximum aesthetics